include/config/rcxml.h

```diff
@@ -80,6 +80,8 @@ struct rcxml {
 		bool outlines;
 		struct wl_list fields;  /* struct window_switcher_field.link */
 	} window_switcher;
+
+	struct wl_list window_rules;
```

Comment:
Missing comment regarding the struct and link member

Reply:
Whoops. Thanks for catching. Done.